### PR TITLE
Add config() example to snowflake warehouse size

### PR DESCRIPTION
**Pull request by patkearns10**
## Description & motivation

Simple addition -- I noticed it only suggests the .yml file way, but doesn't explicitly show the .sql way using a config block. added it in!

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/reference/resource-configs/snowflake-configs.md [hunk 1]
```diff
@@ -182,7 +182,43 @@ models:
 
 ## Configuring virtual warehouses
 
-The default warehouse that dbt uses can be configured in your [Profile](/reference/profiles.yml) for Snowflake connections. To override the warehouse that is used for specific models (or groups of models), use the `snowflake_warehouse` model configuration. This configuration can be used to specify a larger warehouse for certain models in order to control Snowflake costs and project build times.
+The default warehouse that dbt uses can be configured in your [Profile](/reference/profiles.yml) for Snowflake connections. To override the warehouse that is used for specific models (or groups of models), use the `snowflake_warehouse` model configuration. This configuration can be used to specify a larger warehouse for certain models in order to control Snowflake costs and project build times. The following config changes the warehouse for a single model with a config() block.
```
Comment (mirnawong1) [inline website/docs/reference/resource-configs/snowflake-configs.md]:
```suggestion
The default warehouse that dbt uses can be configured in your [Profile](/reference/profiles.yml) for Snowflake connections. To override the warehouse that is used for specific models (or groups of models), use the `snowflake_warehouse` model configuration. This configuration can be used to specify a larger warehouse for certain models in order to control Snowflake costs and project build times. 

The example config below changes the warehouse for a single model with a config() block.
```

File: website/docs/reference/resource-configs/snowflake-configs.md [hunk 2]
```diff
@@ -182,7 +182,43 @@ models:
...
+  
+<File name='models/events/sessions.sql'>
```
Comment (mirnawong1) [inline website/docs/reference/resource-configs/snowflake-configs.md:224]: hey @patkearns10 ,  wanted to see if i can create tabs for the SQL and yml examples so it requires less scrolling for the user experience. see this [slack thread](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1669995969930599)

**Comment by mirnawong1** [comment]
thanks for opening this up and contributing @patkearns10 ! I've left some suggesting and let me know what you think! also waiting to see if i add tabs instead

File: website/docs/reference/resource-configs/snowflake-configs.md (hunk 2 above)
Comment (mirnawong1) [reply website/docs/reference/resource-configs/snowflake-configs.md:224]: ok got it! what about adding the below to the entire yml and SQL file? i can do that if you'd like!

```
`<Tabs
  defaultValue="dbt_project.yml"
  values={[
    { label: 'yml code', value: 'dbt_project.yml', },
    { label: 'sql code', value: 'models/events/sessions.sql', },
  ]
}>

<TabItem value="dbt_project.yml">

<File name='dbt_project.yml'>

``yaml
name: my_project
version: 1.0.0

...

models:
  +snowflake_warehouse: "EXTRA_SMALL"
  my_project:
    clickstream:
      +snowflake_warehouse: "EXTRA_LARGE"

snapshots:
  +snowflake_warehouse: "EXTRA_LARGE"
```

</File>

</TabItem>
<TabItem value="models/events/sessions.sql">

<File name='models/events/sessions.sql'>

```sql
{{
  config(
    materialized='table',
    snowflake_warehouse='EXTRA_LARGE'
  )
}}
with
aggregated_page_events as (
    select
        session_id,
        min(event_time) as session_start,
        max(event_time) as session_end,
        count(*) as count_page_views
    from {{ source('snowplow', 'event') }}
    group by 1
),
index_sessions as (
    select
        *,
        row_number() over (
            partition by session_id
            order by session_start
        ) as page_view_in_session_index
    from aggregated_page_events
)
select * from index_sessions
```

</File>

</TabItem>
</Tabs>`
```

Comment (patkearns10) [reply website/docs/reference/resource-configs/snowflake-configs.md:224]: Sweet! Thanks so much :TIL:

I have made changes and here is the preview: https://deploy-preview-2446--docs-getdbt-com.netlify.app/reference/resource-configs/snowflake-configs

**Review by mirnawong1 — APPROVED** [review]
thanks @patkearns10 - approved for you! feel free to merge, or let me know if you'd like me to merge it for you :D

**Comment by patkearns10** [comment]
Thank you for your help @mirnawong1 !

**Comment by mirnawong1** [comment]
no worries at all :D!

**Pull request merged**